css/style.css

```diff
@@ -16,4 +29,306 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General styles, */
+h1,
+h2,
+h3 {
+  font-weight: 300;
+  line-height: 1.2em;
+  margin: 0;
+}
+
+h1 {
+  font-size: 72px;
+  color: var(--white);
+}
+h2 {
+  font-size: 60px;
+  color: var(--dark-gray);
+}
+h3 {
+  font-size: 40px;
+  color: var(--h3-color);
+}
+
+p {
+  font-size: 25px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: var(--orange);
+  transition: all 0.2s ease-in;
+}
+
+hr {
+  width: 80%;
+  margin-top: 0;
+}
+
+.primary-btn {
+  color: var(--white);
+  background-color: var(--orange);
+  border: none;
+  border-radius: 0.3rem;
+  padding: 1rem 2rem;
+  font-size: 1.2rem;
+  transition: all 0.2s ease-in;
+  text-decoration: none;
+}
+
+.primary-btn:hover {
+  color: var(--orange);
+  background-color: var(--white);
+}
+/* Header styles */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  height: 4rem;
+  padding: 1.5rem 7rem;
+  background-color: var(--lighter-gray);
+}
+
+.header a {
+  display: flex;
+}
+
+.nav-menu {
+  display: flex;
+  align-items: center;
+  gap: 3rem;
+  font-size: 1.3rem;
+}
+
+.nav-menu a {
+  color: var(--light-gray);
+  text-decoration: none;
+}
+
+.nav-menu a:hover {
+  color: var(--orange);
+}
+
+.nav-menu .current-menu-item {
+  color: var(--dark-gray);
+  font-weight: 600;
+}
+
+/* Hero styles */
+
+.hero {
+  --h3-color: var(--white);
+  background: url(../img/first-background.jpg) no-repeat center/cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 700px;
+}
+
+.hero h1 {
+  margin-bottom: 2rem;
+}
+
+.hero h3 {
+  margin-bottom: 2rem;
+}
+
+.hero button {
+  margin: 2rem 0px;
+}
+
+/* Content Styles */
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5rem 0;
+}
+.content h2 {
+  margin: 3rem 0;
+}
+.feature-list {
+  display: flex;
+  gap: 3rem;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 350px;
+}
+
+.feature img {
+  width: 120px;
+}
+
+/* Store Feature Styles */
+
+.store-ft {
+  display: flex;
+  margin-bottom: 5rem;
+}
+
+.store-ft > img {
+  max-width: 40%;
+  flex-grow: 1;
+}
 
+.store-ft-nav {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  max-width: 60%;
+  background-color: var(--light-orange);
+}
+
+.store-ft-nav > h3 {
+  margin: 0 3rem 3rem;
+  font-style: italic;
+}
+
+.color-orange {
+  color: var(--orange);
+}
+
+/* Store page styles */
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-row {
+  display: flex;
+  align-items: center;
+}
+
+.gap-0-5 {
+  gap: 0.5rem;
+}
+
+.store {
+  display: grid;
```

Comment:
I'm glad to see you using grid and flex for diffrent purposes, They make sense where you are using them